snap/snapcraft.yaml

```diff
@@ -282,6 +325,7 @@ apps:
       CONF_ARG: "--confdir=$SNAP_DATA/config/device-virtual"
       PROFILE_ARG: "--profile=res"
       REGISTRY_ARG: "--registry=consul://localhost:8500"
+      Device_ProfilesDir: $SNAP_DATA/config/device-virtual/res
```

Comment:
does this env var make the CONF_ARG and PROFILE_ARG env vars obsolete here?

Reply:
no, it just applies to the directory the profiles are found, where as CONF_ARG and PROFILE_ARG inform the service where the configuration.toml file lives...